bin/cloudformation.py

```diff
@@ -200,7 +229,7 @@ def create_help(header, options):
     config_names = [x.split('/')[1].split('.')[0] for x in glob.glob("configs/*.py") if "__init__" not in x]
     config_help = create_help("config_name supports the following:", config_names)
 
-    actions = ["create", "update", "delete", "post-init", "pre-init", "generate"]
+    actions = ["create", "update", "delete", "post-init", "pre-init", "update-migrate", "generate"]
```

Comment:
Do we want to change the name of the action? Maybe it's easier for the user to just leave it as "update"?

Reply:
The problem is how to determine if they mean run CF update or just the post-update migrations? I could call it `post-update` but I was trying to not take that name if we wanted to support pre-update and post-update like we do pre-init and post-init...

Reply:
Oh I totally misread the change. I thought I saw `update` removed and replaced with `update-migrate`. Is `post-update-migrate` too long? It does give the user a sense of exactly what's happening, though.

Reply:
Not sure. @sandyhider / @lrodri29 any input into the command name?